### PR TITLE
Us57

### PR DESCRIPTION
**Pull request by LudovicRAVINET**
problem when session expired (24 mn maximum lifetime session):  I had a test to verify a session is existing before destroying it.
and I added a session_start in each function of the controller.
I think perhaps we should have a global session manager for our project because of this maximum lifetime session. Perhaps an other table Session in database?

### CODE REVIEW

File: src/Controller/SoughtProfileController.php [hunk 1]
```diff
@@ -21,6 +24,7 @@ public function resultResearch()
 
     public function matchUpdate(int $searchingId, int $foundId, string $statusSearching)
     {
+        session_start();
```
Comment (aldaitzwild) [inline src/Controller/SoughtProfileController.php:27]: No if you need a session_start() on every page, put it in index.php. The only time it's ok to have a session_start() in a controller it's after a session_destroy().

File: src/Controller/SoughtProfileController.php [hunk 2]
```diff
@@ -35,6 +39,7 @@ public function matchUpdate(int $searchingId, int $foundId, string $statusSearch
 
     public function profileDisplay()
     {
+        session_start();
```
Comment (aldaitzwild) [inline src/Controller/SoughtProfileController.php:42]: Same remark.

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/Controller/SoughtProfileController.php (hunk 1 above)
Comment (LudovicRAVINET) [reply src/Controller/SoughtProfileController.php]: ok, I put session_start() with a test at the top of index.php

File: src/Controller/SoughtProfileController.php (hunk 2 above)
Comment (LudovicRAVINET) [reply src/Controller/SoughtProfileController.php]: ok, I put session_start() with a test at the top of index.php

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**